control-plane/src/values.js

```diff
@@ -0,0 +1,23 @@
+// ALL THE VALUES THAT ARE COMMONLY USED THROUGHOUT THE FILES
+const collection = process.env.NODE_ENV === 'production' ? 'jobs' : 'jobs-test'
+
+export default {
+    currentLevelsOne: 0,
```

Comment:
What are these for? Is there a way to have this more dynamic? This sounds like it will only work for LevelsOne and LevelsTwo?

Reply:
This was related to adding jobs. So I have removed that